flink-state-backends/flink-statebackend-rocksdb/src/test/java/org/apache/flink/state/rocksdb/RocksDBWriteBatchWrapperTest.java

```diff
@@ -83,6 +85,7 @@ public void testAsyncCancellation() throws Exception {
                                 batchSizeBytes)) {
             registry.registerCloseable(writeBatchWrapper.getCancelCloseable());
             writeStartedFuture.complete(null);
+            closeCompleteFuture.join();
```

Comment:
Why not use `cancellationRequestedFuture` here?

Reply:
You're right @Zakelly ! I've updated the PR to use only `cancellationRequestedFuture` now. Please take a look again. Thanks.